iset.mm

```diff
@@ -99583,6 +99597,769 @@ seq K ( .+ , G , S ) ) $=
+          IUBTZVJFUKZCVJUCDVJUCVAVBULAVJVJFUMVKKVJVJFUNUOLMVJVJCDFUPUQUR $.
+      $}
+    $}
+  $}
+
+  ${
+    iseqf1olemqf.k $e |- ( ph -> K e. ( M ... N ) ) $.
+    iseqf1olemqf.j $e |- ( ph
+      -> J : ( M ... N ) -1-1-onto-> ( M ... N ) ) $.
+    iseqf1olemqf.q $e |- Q = ( u e. ( M ... N ) |->
+      if ( u e. ( K ... ( `' J ` K ) ) ,
+        if ( u = K , K , ( J ` ( u - 1 ) ) ) ,
+        ( J ` u ) ) ) $.
+
+    ${
+      $d J u x $.  $d K u x $.  $d M u x $.  $d N u $.  $d Q x $.  $d ph u x $.
+      $( Lemma for ~ iseqf1o .  Domain and codomain of ` Q ` .  (Contributed by
+         Jim Kingdon, 26-Aug-2022.) $)
+      iseqf1olemqf $p |- ( ph
+          -> Q : ( M ... N ) --> ( M ... N ) ) $=
+        ( cfz co cv ccnv cfv wcel wceq c1 cif adantr cmin wa wf1o iseqf1olemqcl
+        simpr fmptd ) ABFGKLZBMZEEDNOKLPUHEQEUHRUALDOSUHDOSUGCAUHUGPZUBUHDEFGAE
+        UGPUIHTAUGUGDUCUIITAUIUEUDJUF $.
+    $}
+
+    ${
+      $d A u $.  $d B u $.  $d J u x $.  $d K u x $.  $d M u x $.  $d N u $.
+      $d Q x $.  $d ph x $.
+      iseqf1olemmo.a $e |- ( ph -> A e. ( M ... N ) ) $.
+      iseqf1olemmo.b $e |- ( ph -> B e. ( M ... N ) ) $.
+      iseqf1olemmo.eq $e |- ( ph -> ( Q ` A ) = ( Q ` B ) ) $.
+      $( Lemma for ~ iseqf1o .  Showing that ` Q ` is one-to-one.  (Contributed
+         by Jim Kingdon, 27-Aug-2022.) $)
+      iseqf1olemmo $p |- ( ph -> A = B ) $=
+        ( wcel wn wa ad2antrr syl ccnv cfv cfz co wceq wf1o simplr iseqf1olemab
+        simpr jca iseqf1olemnab pm2.21dd wo wdc cz elfzelz f1ocnv f1of ffvelrnd
+        wf 3syl fzdcel syl3anc exmiddc adantr mpjaodan eqcomd iseqf1olemnanb )
+        ACGGFUAZUBZUCUDZPZCDUEZVLQZAVLRZDVKPZVMVPQZVOVPRBCDEFGHIAGHIUCUDZPZVLVP
+        JSAVRVRFUFZVLVPKSACVRPZVLVPMSADVRPZVLVPNSACEUBZDEUBZUEZVLVPOSLAVLVPUGVO
+        VPUIUHVOVQRZVLVQRZVMWFVLVQAVLVQUGVOVQUIUJAWGQVLVQABCDEFGHIJKMNOLUKSULAV
+        PVQUMZVLAVPUNZWHADUOPZGUOPZVJUOPZWIAWBWJNDHIUPTAVSWKJGHIUPTZAVJVRPWLAVR
+        VRGVIAVTVRVRVIUFVRVRVIUTKVRVRFUQVRVRVIURVAJUSVJHIUPTZDGVJVBVCVPVDTZVEVF
+        AVNRZVPVMVQWPVPRZVPVNRZVMWQVPVNWPVPUIAVNVPUGUJAWRQVNVPABDCEFGHIJKNMAWCW
+        DOVGLUKSULWPVQRBCDEFGHIAVSVNVQJSAVTVNVQKSAWAVNVQMSAWBVNVQNSAWEVNVQOSLAV
+        NVQUGWPVQUIVHAWHVNWOVEVFAVLUNZVLVNUMACUOPZWKWLWSAWAWTMCHIUPTWMWNCGVJVBV
+        CVLVDTVF $.
+    $}
+
+    ${
+      $d J u x $.  $d K u x $.  $d K v w x $.  $d M u x $.  $d M v w x $.
+      $d N u x $.  $d N v w x $.  $d Q v w x $.  $d ph u x $.  $d ph v w x $.
+      $d u v w $.
+      $( Lemma for ~ iseqf1o . ` Q ` is a permutation of ` ( M ... N ) ` .
+         ` Q ` is formed from the constant portion of ` J ` , followed by the
+         single element ` K ` (at position ` K ` ), followed by the rest of J
+         (with the ` K ` deleted and the elements before ` K ` moved one
+         position later to fill the gap).  (Contributed by Jim Kingdon,
+         21-Aug-2022.) $)
+      iseqf1olemqf1o $p |- ( ph
+          -> Q : ( M ... N ) -1-1-onto-> ( M ... N ) ) $=
```

Comment:
fits on single line

etc.

Reply:
Will remove.

As for the "etc" I combined a few more - hopefully I got enough of them to deal with the worst of this.